cwms-data-api/src/test/resources/cwms/cda/data/dto/project_children.json

```diff
@@ -0,0 +1,48 @@
+{
+  "project": {
+    "office-id": "SPK",
+    "name": "TestProject"
+  },
+  "embankments": [
```

Comment:
what do you think about just using the location kind here for the variable name instead of the plural? 
"embankment": [
...

Reply:
It holds a list so I went with plural but it could be singular or the actual kind "EMBANKMENT".  Is there a reason you prefer the singular?

Reply:
If someone queries locations and uses the kind then tries to map the kind to here, they can't without some custom handling to understand that the sets are plural here.

Correspondingly, take the results of this query then match the project embankments against the location catalog filtered to locations of kind embankment.

Reply:
Well, it's verbose, but it's abundantly clear.